compiler/qsc_linter/src/lints/ast.rs

```diff
@@ -6,6 +6,23 @@ use crate::linter::ast::declare_ast_lints;
 use qsc_ast::ast::{BinOp, ExprKind, Lit, StmtKind};
 use qsc_data_structures::span::Span;
 
+// Read Me:
```

Comment:
Should these be `///` docstrings?

Reply:
No, unfortunately I get a compiler warning if I try to make those `///` docstrings.